host.go

```diff
@@ -522,7 +519,7 @@ func (h *gpbftRunner) Progress() (instance, round uint64, phase gpbft.Phase) {
 // These will be used as input to a subsequent instance of the protocol.
 // The chain should be a suffix of the last chain notified to the host via
 // ReceiveDecision (or known to be final via some other channel).
-func (h *gpbftHost) GetProposal(instance uint64) (_ *gpbft.SupplementalData, _ gpbft.ECChain, _err error) {
+func (h *gpbftRunner) GetProposal(instance uint64) (_ *gpbft.SupplementalData, _ gpbft.ECChain, _err error) {
```

Comment:
any reason to change this?

Reply:
Because it was a nightmare navigating between runner and host, when they are essentially the same. Without this change one has to cast the runner to broadcast etc. etc. 

I am willing to be convinced this type separation is necessary.

Reply:
No preference. The only reason we had the split was to make it clear which methods were gpbft-facing and which were client-facing.

Reply:
> to make it clear which methods were gpbft-facing and which were client-facing

For what it's worth I as a maintainer find it confusing and too expensive at the price of casting.

Reply:
Two casts? I don't think it added much anyways (the two types was a replacement for two entirely separate structs), but this kind of newtyping really isn't that uncommon.

Reply:
Since this is two votes against one, i will revert this and cast.

Reply:
> this kind of newtyping really isn't that uncommon.

It is my opinion that this practice is awful.

Reply:
Eh, I've gotten used to it at this point.

Reply:
But note: I'm not going complain if you want to re-do this change in a new PR.